### ISSUE TITLE
ScannerError while testing CFME-5.7.1 with the framework

### ISSUE THREAD

**Issue by hroyrh** [redhat-performance/cfme-performance#4]
I faced ScannerError issue when running workload test with CFME-5.7.1  appliance using the framework. Here is details regarding the issue : https://gist.github.com/hroyrh/56363764f0b1ec5ab058b6e59a37cef1
The framework was working for CFME-5.7.0.x , but it is not working for the new version. Need to add support for tests for the latest version

**Comment by arcolife** [comment]
@hroyrh could you correct the pastebin reference from RH internal to.. gist.github.com ? 

Also, add more details about the bug here? thanks.

**Comment by arcolife** [comment]
updates:

```
[root@CF-B2B-R0000-57100-test vmdb]# ./bin/rails console
/var/www/miq/vmdb/app/models/mixins/supports_feature_mixin.rb:103: warning: key :terminate is duplicated and overwritten on line 111
Loading production environment (Rails 5.0.0.1)
```
While running rails console on 5.7.1.0, we get the aforementioned warning and as a result, the stderr is redirected to the output which is piped into the remote ssh_client's response string (yaml_load) and when this is loaded through yaml ( https://gist.github.com/hroyrh/56363764f0b1ec5ab058b6e59a37cef1#file-scannererror-L43 ), this produces a format anomaly.

We need to handle this exception here.

**Comment by arcolife** [comment]
bugzilla link: https://bugzilla.redhat.com/show_bug.cgi?id=1420284

**Comment by arcolife** [comment]
fixed by #7
